lint-checks-android/src/main/java/com/uber/lintchecks/android/XmlWebViewInsideScrollViewDetector.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2019. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.lintchecks.android
+
+import com.android.SdkConstants.ANDROID_NS_NAME_PREFIX
+import com.android.SdkConstants.ATTR_FILL_VIEWPORT
+import com.android.resources.ResourceFolderType
+import com.android.tools.lint.detector.api.Category
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.ResourceXmlDetector
+import com.android.tools.lint.detector.api.XmlContext
+import com.android.tools.lint.detector.api.Severity
+import org.w3c.dom.Element
+
+/** Custom lint check to make sure that Scrollview which has WebView as its child sets fillViewport to true*/
+class XmlWebViewInsideScrollViewDetector : ResourceXmlDetector() {
+    companion object {
```

Comment:
Can you change indent to 2 spaces instead of 4? I think spotless covers this but not sure why it didn't :/

Reply:
Sure will do